concordium-node/src/plugins/consensus.rs

```diff
@@ -311,38 +336,45 @@ fn send_msg_to_consensus(
     source_id: RemotePeerId,
     consensus: &ConsensusContainer,
     message: &ConsensusMessage,
-) -> anyhow::Result<ConsensusFfiResponse> {
+) -> anyhow::Result<(ConsensusFfiResponse, Option<ExecuteBlockCallback>)> {
```

Comment:
I haven't considered this too much, my initial thought was that `ResultSuccess` is rather generic in its nature. I will check up on this.

Reply:
I would prefer to keep as is for the above reason. 
I would like to avoid introducing an abstraction layer for this "special" case. By "special" I mean, that in particular `ResultSuccess` is used also for e.g. transactions and finalization records. To properly support that I think a new type is required (which anyway always would be a noop).

Reply:
But maybe I'm wrong.